test/opt/local_single_store_elim_test.cpp

```diff
@@ -1211,80 +1211,6 @@ TEST_F(LocalSingleStoreElimTest, DebugValueTest) {
   SinglePassRunAndMatch<LocalSingleStoreElimPass>(text, false);
 }
 
-TEST_F(LocalSingleStoreElimTest, DebugDeclareForFunctionParameter) {
```

Comment:
Why is the whole test deleted?

Reply:
I added this unit test in #3923 but I revert the commit from #3923 in this PR because I found the idea was wrong (it cannot handle all the cases and it has a bug - generates some illegal forward references).

Deleting this unit test is a part of the revert.